packages/ember-runtime/lib/mixins/target_action_support.js

```diff
@@ -144,5 +148,17 @@ function getTarget(instance) {
     }
   }
 
+  // if a `targetObject` CP was provided, use it
+  if (target) { return target; }
+
+  // if _targetObject use it
+  if (instance._targetObject) { return instance._targetObject; }
+
   return null;
 }
+
+function _deprecateTargetObject() {
+  let message = 'Usage of `targetObject` is deprecated. Please use `target` instead.';
```

Comment:
I think we should pass a bit more debugging info here.  If I saw this message in the console, I'm not sure how I would track down where it was coming from.  Perhaps you can include `this` (which will display something like `<component:foo-bar#1234>` which might give someone a clue...).